src/dynamicprompts/wildcards/wildcard_manager.py

```diff
@@ -141,10 +155,18 @@ def get_all_values(self, wildcard: str) -> list[str]:
                     ", ".join(str(coll) for coll in rec_colls),
                 )
 
-        sorted_values = sorted(values)
+        list_values = values
+        if self._dedup_wildcards:
+            d = {v: None for v in list_values}
+            list_values = list(d.keys())
```

Comment:
I think
```suggestion
            list_values = list(dict.fromkeys(list_values, None))
```
would do the trick.